pcsx2/ps2/BiosTools.h

```diff
@@ -14,6 +14,9 @@
  */
 
 #pragma once
+// Uncomment this to use hardcoded addresses for the thread view in the debugger
+// It supports the bioses _only_ hardcoded, nothing more 
+//#define USE_HARDCODED_BIOS_INFORMATION
```

Comment:
We don't really need to add this here, it should be done at the project level, eg adding the use flag to cmake/VS

Reply:
you can go ahead and remove the old way then, we can always revert